shentu/chain.json

```diff
@@ -35,47 +35,38 @@
     "genesis": {
       "genesis_url": "https://raw.githubusercontent.com/ShentuChain/mainnet/main/shentu-2.2/genesis.json"
     },
-    "recommended_version": "v2.7.1",
+    "recommended_version": "v2.8.0",
```

Comment:
Please add v2.8.0 to the bottom of the `versions` array below

Reply:
added.